projects/js-packages/publicize-components/src/social-store/resolvers.js

```diff
@@ -75,11 +75,7 @@ export function getConnections() {
 		const connections = editor.getEditedPostAttribute( 'jetpack_publicize_connections' );
 
 		dispatch( setConnections( connections || [] ) );
-
-		// For now, we are interested in the initial connections only if the post is scheduled.
-		if ( editor.isCurrentPostScheduled() ) {
-			dispatch( setInitialConnections( connections || [] ) );
-		}
+		dispatch( setInitialConnections( connections || [] ) );
```

Comment:
Whether to subtract the initial connections count is dependent on the post date, and as that can be edited, I moved it to the useScheduledPost hook. The problem is that we need to do different things based on if the post date moves in or out of the 30 day window.

I think we might be able to solve it by storing the initial date in a ref and seeing if the edited date is moving in or out of the 30 day window.